api/holiday_acres_api/models.py

```diff
@@ -4,28 +4,34 @@
 class User(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
-    name = models.CharField(max_length=200)
+    # USER DATA
+    first_name = models.CharField(max_length=200)
+    last_name = models.CharField(max_length=200)
+    email = models.CharField(max_length=200)
 
 
 class Paddock(models.Model):
-    name = models.CharField(max_length=200)
-    paddock_name = models.CharField(max_length=200)
-    horse = models.CharField(max_length=200)
-    tier = models.IntegerField()
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+    # PADDOCK DATA
+    paddock_name = models.CharField(max_length=200)
+    paddock_tier = models.IntegerField()
 
 
 class Horse(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    # DB RELATIONSHIPS
+    # one-to-many relation (one paddock to many horses)
+    paddock = models.ForeignKey(
+        Paddock, related_name="horses", on_delete=models.CASCADE
```

Comment:
I'm not sure a `CASCADE` delete makes sense here. If I understand correctly -- if you deleted the `Horse` this would also delete the `Paddock` model that this is associated with.